main.tf

```diff
@@ -540,12 +547,43 @@ resource "aws_appautoscaling_policy" "ecs_service" {
   service_namespace  = aws_appautoscaling_target.ecs_service[0].service_namespace
 
   target_tracking_scaling_policy_configuration {
```

Comment:
Er å ha både predefined og customized lov eller noe som gir mening?

Hvis ikke tror jeg at her er et godt sted å bruke en lifecycle precondition for å kun tillate at et av argumentene eksisterer.

https://developer.hashicorp.com/terraform/language/expressions/custom-conditions#preconditions-and-postconditions

Reply:
Ser ut som enten eller